<robot name="mara">
	<link name="base_link">
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<box size=".07125 .07125 0.01"/>
			</geometry>
			<material name="Cyan1">
					<color rgba="0 0.9 0.9 1.0"/>
			</material>
		</visual>	
	</link>

	<link name="link1" >
		<visual>
			<origin xyz="0 0 0.224" rpy="0 0 0"/>
			<geometry>
				<cylinder radius=".07125" length=".448"/>
			</geometry>
			<material name="Cyan1">
					<color rgba="0 0.9 0.9 1.0"/>
			</material>
		</visual>	
	</link>

	<link name="link2" >
		<visual>
			<origin xyz="0.0 0 0.205" rpy="0 0 0"/>
			<geometry>
				<cylinder radius=".04" length=".41"/>
			</geometry>
			<material name="Pink">
					<color rgba="0.8 0.0 0.9 1.0"/>
			</material>
		</visual>	
	</link>

	<link name="link3" >
		<visual>
			<origin xyz="0.0 0 0.076" rpy="3.14 0 0"/>
			<geometry>
				<cylinder radius=".03" length=".152"/>
			</geometry>
			<material name="White2">
					<color rgba="1 0.9 0.9 1.0"/>
			</material>
		</visual>	
	</link>
	<link name="link4" >
		<visual>
			<origin xyz="0 0 0.076" rpy="0 0 0"/>
			<geometry>
				<cylinder radius=".03" length=".152"/>
			</geometry>
			<material name="White3">
					<color rgba="1 0.0 0.0 1"/>
			</material>
		</visual>	
	</link>
	<link name="link5" >
		<visual>
			<origin xyz="0 0 0.025" rpy="0 0 0"/>
			<geometry>
				<cylinder radius=".02" length=".05"/>
			</geometry>
			<material name="Blue1">
					<color rgba="0.0 0.2 0.9 0"/>
			</material>
		</visual>	
	</link>
	<link name="link6" >
		<visual>
			<origin xyz="0 0 0" rpy="0 1.57 0"/>
			<geometry>
				<cylinder radius=".016" length=".05"/>
			</geometry>
			<material name="Blue2">
					<color rgba="0.0 1 0.9 0"/>
			</material>
		</visual>	
	</link>

	<joint name="joint1" type="continuous">
		<parent link="base_link"/>
		<child link="link1"/>
		<origin xyz="0 0 0" rpy="0 0 -0.785"/>
                <axis xyz="0 0 1" />
	</joint>

	<joint name="joint2" type="continuous">
		<parent link="link1"/>
		<child link="link2"/>
		<origin xyz="0.2 0.0 0.448" rpy="1.57 0 0"/>
                <axis xyz="1 0 0" />
	</joint>

	<joint name="joint3" type="continuous">
		<parent link="link2"/>
		<child link="link3"/>
		<origin xyz="-0.08 0 0.41" rpy="0 0 0"/>
                <axis xyz="1 0 0" />
	</joint>

	<joint name="joint4" type="continuous">
		<parent link="link3"/>
		<child link="link4"/>
		<origin xyz="0.00 0 0.152" rpy="0 0 0"/>
                <axis xyz="0 0 1" />
	</joint>

	<joint name="joint5" type="continuous">
		<parent link="link4"/>
		<child link="link5"/>
		<origin xyz="0 0.0 0.152" rpy="0 0 0"/>
                <axis xyz="0 1 0" />
	</joint>

	<joint name="joint6" type="continuous">
		<parent link="link5"/>
		<child link="link6"/>
		<origin xyz="0 0.0 0.05" rpy="0 0 0"/>
                <axis xyz="0 0 1" />
	</joint>
</robot>